news/2024/2024-09-22-project-loved-seven-years-september-2024.md

```diff
@@ -0,0 +1,478 @@
+
+[![Project Loved website banner](/wiki/shared/news/banners/project-loved/banner-website.png)](https://loved.sh/)
+
+Thank you for voting on the August round everyone! To view the results, click one of the following links:
+
+* [Results for osu!](https://osu.ppy.sh/community/forums/topics/1966748)
+* [Results for osu!taiko](https://osu.ppy.sh/community/forums/topics/1966747)
+* [Results for osu!catch](https://osu.ppy.sh/community/forums/topics/1966746)
+* [Results for osu!mania](https://osu.ppy.sh/community/forums/topics/1966745)
+
+Did you know we have dedicated Project Loved community servers for each game mode? Check them out here!
+
+[![osu! Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-osu.png)](https://discord.gg/gn58Uk5sTE)
+
+[![osu!taiko Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-taiko.png)](https://discord.gg/BuJ6TVPmtN)
+
+[![osu!catch Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-catch.png)](https://discord.gg/phgtyS4UCh)
+
+[![osu!mania Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-mania.png)](https://discord.gg/Ededv7m)
+
+Before we get stuck in, we want to thank everyone for sticking with us for **7 years**!
+
+September 2024 marks Project Loved's 7th birthday, and we couldn't have done it all without the incredible support from the community and our amazing [volunteer contributors](https://loved.sh/contributors). Please send them your love!
+
+To mark this momentous occasion, why not celebrate by showing some extra love to your favourite creators, and if you feel someone's missing out on that shiny Loved beatmap, remember that [submissions](https://loved.sh/submit) are always open!
+
+Lastly, we have some team updates incoming! This is what's hot off the press:
+
+* [aceticke](https://osu.ppy.sh/users/8838763) (that's me!) has resigned from osu!taiko captain and metadata reviewer duties. I'll be focusing on coordination going forward!
+* [Jerry](https://osu.ppy.sh/users/605973) has resigned as a metadata reviewer, bid him farewell and a good luck on his other ventures!
+
+<div align="center">
+  <video width="95%" controls>
+    <source src="https://assets.ppy.sh/project-loved/2024/sep24-intro.mp4" type="video/mp4" preload="none">
+  </video>
+</div>
+
+### Navigation
+
+- **[osu!](#osu!)** ([Download pack](https://packs.ppy.sh/LR12%20-%20Project%20Loved:%20September%202024%20(osu!).zip))
+- **[osu!taiko](#osu!taiko)** ([Download pack](https://packs.ppy.sh/LR11%20-%20Project%20Loved:%20September%202024%20(osu!taiko).zip))
+- **[osu!catch](#osu!catch)** ([Download pack](https://packs.ppy.sh/LR10%20-%20Project%20Loved:%20September%202024%20(osu!catch).zip))
+- **[osu!mania](#osu!mania)** ([Download pack](https://packs.ppy.sh/LR9%20-%20Project%20Loved:%20September%202024%20(osu!mania).zip))
```

Comment:
i recommend reordering like this

```suggestion
### Navigation

- **[osu!](#osu!)** ([Download pack](https://packs.ppy.sh/LR12%20-%20Project%20Loved:%20September%202024%20(osu!).zip))
- **[osu!taiko](#osu!taiko)** ([Download pack](https://packs.ppy.sh/LR11%20-%20Project%20Loved:%20September%202024%20(osu!taiko).zip))
- **[osu!catch](#osu!catch)** ([Download pack](https://packs.ppy.sh/LR10%20-%20Project%20Loved:%20September%202024%20(osu!catch).zip))
- **[osu!mania](#osu!mania)** ([Download pack](https://packs.ppy.sh/LR9%20-%20Project%20Loved:%20September%202024%20(osu!mania).zip))

## Summary

<div align="center">
  <video width="95%" controls>
    <source src="https://assets.ppy.sh/project-loved/2024/sep24-intro.mp4" type="video/mp4" preload="none">
  </video>
</div>
```

Reply:
@cl8n Will need changes to project-loved to format it like this automatically.